doc/start/hardware-recommendations.rst

```diff
@@ -143,23 +145,36 @@ write times, as well as total throughput. These physical limitations affect
 overall system performance--especially during recovery. We recommend using a
 dedicated (ideally mirrored) drive for the operating system and software, and
 one drive for each Ceph OSD Daemon you run on the host (modulo NVMe above).
-Many "slow OSD" issues not attributable to hardware failure arise from running
-an operating system and multiple OSDs on the same drive. Since the cost of troubleshooting performance issues on a small cluster likely exceeds the cost of the extra disk drives, you can optimize your cluster design planning by avoiding the temptation to overtax the OSD storage drives.
+Many "slow OSD" issues (when they are not attributable to hardware failure)
+arise from running an operating system and multiple OSDs on the same drive.
+Because it usually costs more to troubleshoot performance issues on a small
+cluster than it costs to buy extra disk drives, optimal cluster design likely
+involves adding more disk drives in order to avoid overtaxing the OSD storage
+drives.
 
-You may run multiple Ceph OSD Daemons per SAS / SATA drive, but this will likely
-lead to resource contention and diminish the overall throughput. 
+It is technically possible to run multiple Ceph OSD Daemons per SAS / SATA
+drive, but this will likely lead to resource contention and diminish the
+overall throughput.
+
+It is technically possible to store journal data and object data on the same
+drive, but it is not recommended. Storing journal data and object data on the
+same drive increases the time taken to journal a write and an ACK to the client
+because Ceph must write to the journal before it can ACK the write.
+
+To get the best performance out of Ceph, (1) operating systems, (2) OSD data,
+and (3) OSD journals should be run on separate drives.
```

Comment:
Were we removing references to Filestore journals? "Metadata" or "WAL+DB" instead?

Reply:
To be continued in a future PR.